lib/install/management/finder.go

```diff
@@ -111,11 +111,18 @@ func (d *Dispatcher) NewVCHFromComputePath(computePath string, name string, v *v
 	if vchPool == nil {
 		vchPool, err = d.session.Finder.ResourcePool(d.op, d.vchPoolPath)
 		if err != nil {
-			d.op.Errorf("Failed to get VCH resource pool %q: %s", d.vchPoolPath, err)
-			return nil, err
+			// we didn't find the ResourcePool with a name matching the appliance, so
```

Comment:
I'm unclear why this is here and do not find the comment enlightening. I'm guessing it's because we don't have resource pools with DRS off, but then we're still looking for resource pools for some reason...

Why not use the same logic as in `create.go` to set the vchPoolPath and be explicit about what we expect up front.

Reply:
I wanted to make as few changes as possible to these finder funcs -- they need to be rewritten and not rely on inventory path, etc...but that change was fairly large...so I backed out of it and went with a minimalist approach.